docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -33,7 +33,31 @@ Keeping informed
 To track changes that affect collections:
 
 * Join the `Collection Maintainers & Contributors forum group <https://forum.ansible.com/g/CollectionMaintainer>`_.
-* Subscribe to the `Bullhorn <https://forum.ansible.com/c/news/bullhorn/17>`_ Ansible contributor newsletter.
+* Subscribe to the `Bullhorn <https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn>`_ Ansible contributor newsletter.
```

Comment:
Shouldn't use this a proper RST reference, since this is part of the same docsite? (You'd have to add a label to `community/communication.rst` since right now there is none yet.)

Reply:
should be already there AFAIK. this reference moved to the doc from that old location, thanks for spotting, i'll fix it

Reply:
ah, nope, you're right, adding the anchor, thanks